addons/base_onboarding/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Onboarding toolbox',
+    'version': '1.0',
+    'category': 'Hidden',
+    'description': """
+This module handles onboarding progress
+================================================================================
+
+Shows you a list of applications features to install from.
+
+    """,
+    'depends': ['base'],
+    'installable': True,
+    'auto_install': True,
```

Comment:
Is it mandatory to make it auto_install ? Can't we add it in dependencies of appointment ?